app/src/main/java/com/example/movopfy/features/movies/presentation/ui/MoviesScreen.kt

```diff
@@ -1,14 +1,50 @@
 package com.example.movopfy.features.movies.presentation.ui
 
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.grid.rememberLazyGridState
+import androidx.compose.material3.Scaffold
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
 import androidx.compose.ui.Modifier
 import androidx.navigation.NavController
+import com.example.movopfy.features.movies.presentation.viewmodel.MoviesViewModel
+import com.example.movopfy.uiComponents.components.ProgressBarLoading
+import org.koin.androidx.compose.koinViewModel
 
 @Composable
 fun MoviesScreen(
     modifier: Modifier = Modifier,
     navController: NavController,
-    category: String
+    category: String,
+    viewModel: MoviesViewModel = koinViewModel()
 ) {
+    val uiState by viewModel.uiState.collectAsState()
 
+    LaunchedEffect(Unit) {
+        viewModel.getKinopoiskList(category = category)
```

Comment:
Why you do it here? It can be done in `init {}` of viewModel

Reply:
I cannot it should be parameters count

Reply:
but is should put category from screen parameters in request

Reply:
Okey, I didn't see